content/tutorials/generic/bootloader-reloading/bootloader-reloading.md

```diff
@@ -0,0 +1,170 @@
+- Select **Sketch > Upload** from the Arduino IDE's menu.
+- Wait for the upload to finish successfully.
+- Unplug the **programmer** Arduino board from your computer.
+- Plug the SD card into the SD slot connected to your Arduino board.
+- Connect the **programmer** Arduino board to the target Arduino board as follows:
+
+    | Programmer  | Target      |
+    | ----------- | ----------- |
+    | VCC         | +3V3        |
+    | 12          | SWDIO       |
+    | 11          | SWCLK       |
+    | GND         | GND         |
+    | 9           | RESETN      |
+
+    SWD pads on MKR boards other than MKR 1000:
+
+    ![](assets/SWDpadsMKR.png)
+
+    MKR1000 SWD header pinout:
+
+    ![](assets/SWDMKR1000header.png)
+
+    Nano 33 IoT SWD pads:
+
+    ![](assets/SWDpadsNano33IoT.png)
+
+- Plug the USB cable of the **programmer** Arduino board into your computer.
+- Select **Tools > Serial Monitor** from the Arduino IDE's menu. You should now see the Serial Monitor output showing the **target** board detected, and the bootloader file flashed to it successfully.
+- Unplug the **programmer** Arduino board from your computer.
+- Disconnect the programmer Arduino board from the target Arduino board.
+
+**Congratulations** you have now successfully burned the bootloader on your Arduino board and can go back to upload sketches like before.
+
+## No SD card
+### Requirements
+- An extra SAMD-based Arduino board to use as the programmer. 
+- A way to make connections to the SWD pins on your target Arduino board.
+
+### Instructions
+- Plug the USB cable of the Arduino board you will be using as a programmer into your computer.
+- Select **Sketch > Include Library > Manage  Libraries…** from the Arduino IDE menu.
+- Wait for the index updates to finish.
+- In the "**Filter your search…**” field type ‘Adafruit DAP library’.
+- Press **Enter**.
+- Click on “**Adafruit DAP library by Adafruit**”.
+- Click the **Install** button.
+- You may now get a dialog asking whether to install the library's dependencies. If so, click the "**Install All**" button.
+- Wait for the installation to finish
+- Close the **Library Manager**
+- Select **File > Example > Adafruit DAP Library > samd21 > flash_MKR_bootloaders** from the Arduino IDE menu. (despite the “MKR” in the sketch name, this also supports the **Nano 33 IoT**).
+- Select your **programmer** Arduino board from the IDE’s **Tools > Board** menu.
+- Select the port of the **programmer** Arduino board from the Arduino IDE’s Tools > Port menu.
+- Select **Sketch > Upload** from the Arduino IDE menu.
+- Unplug the **programmer** Arduino board from your computer.
+- Connect the **programmer** Arduino board to the target Arduino board as follows:
+
+    | Programmer  | Target      |
+    | ----------- | ----------- |
+    | VCC         | +3V3        |
+    | 12          | SWDIO       |
+    | 11          | SWCLK       |
+    | GND         | GND         |
+    | 9           | RESETN      |
```

Comment:
```suggestion
    | 1           | SWDIO       |
    | 2           | SWCLK       |
    | GND         | GND         |
    | 0           | RESETN      |
```

Unfortunately they chose different pins to use in the two example sketches. This example uses pins 1, 2, 0:

https://github.com/adafruit/Adafruit_DAP/blob/1.8.0/examples/samd21/flash_MKR_bootloaders/flash_MKR_bootloaders.ino#L4-L6